pandas/tests/base/test_misc.py

```diff
@@ -142,6 +142,31 @@ def test_memory_usage_components_narrow_series(any_real_numpy_dtype):
     assert total_usage == non_index_usage + index_usage
 
 
+def test_memory_usage_doesnt_trigger_engine(index):
```

Comment:
Could you move this test to `pandas/tests/indexes/test_old_base.py`

Reply:
Done